firefly_client/firefly_client.py

```diff
@@ -277,7 +287,24 @@ def get_firefly_url(self, mode='full', channel=None):
 
         return url + channel
 
-    def launch_browser(self, url=None, channel=None, force=False):
+    def display_url(self):
+        """
+        Display URL in a user-friendly format
+
+        """
+        try:
+            ipy_str = str(type(get_ipython()))
+            if 'zmqshell' in ipy_str:
+                from IPython.display import display, Markdown
+                display(
+                    Markdown('>Open your web browser to [{}](<a href={} target="_blank">{}</a>)'
+                                 .format([self.get_firefly_url()]*3)))
+                return
```

Comment:
It will add Markdown output in the notebook context. In IPython it will just print text. 

If `get_ipython` raises an Exception, the context is a plain Python session. If `get_ipython` exists, then the return value is examined to determine whether it is an IPython session or a Jupyter notebook.